alacritty_terminal/src/term/search.rs

```diff
@@ -12,47 +17,46 @@ use crate::term::Term;
 /// Used to match equal brackets, when performing a bracket-pair selection.
 const BRACKET_PAIRS: [(char, char); 4] = [('(', ')'), ('[', ']'), ('{', '}'), ('<', '>')];
 
+/// Maximum DFA size to prevent pathological regexes taking down the entire system.
+const MAX_DFA_SIZE: usize = 100_000_000;
```

Comment:
How was it computed?

Reply:
It wasn't computed. I just picked a random number, checked how long it would hang Alacritty up for, then decided that was acceptable.